specification/common-types/resource-management/v1/privatelinks.json

```diff
@@ -0,0 +1,162 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "PrivateEndpoint": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ARM identifier for Private Endpoint"
+        }
+      },
+      "description": "The Private Endpoint resource."
+    },
+    "PrivateEndpointConnection": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "x-ms-client-flatten": true,
+          "description": "Resource properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The Private Endpoint Connection resource."
+    },
+    "PrivateEndpointConnectionProperties": {
```

Comment:
Actually I am wondering for private endpoint connection properties, why group id (linked sub-resource, e.g. file, blob...) is not provided.  @priyapansrp do you have any idea?

Reply:
I am not following. This is supposed to be common private endpoint at service level, As we lift them into common folder, no RP specific resources should be here. Nor I am sure what file, blobs are related to private link endpoints. Waiting for @priyapansrp for response here.

Reply:
The groupid is exposed to private endpoint properties and not private endpoint connection. I think it makes sense to reflect groupid on private endpoint connection as well. Let me check with Mario (PM for the feature) as well.

Reply:
GroupId is on PrivateLinkResourceProperty. See line 116 below.